src/core/curve.py

```
@@ -103,8 +108,8 @@ async def get_curve_state(self, curve_address: Pubkey) -> BondingCurveState:
             return BondingCurveState(account.data)
 
         except Exception as e:
-            logger.error(f"Failed to get curve state: {str(e)}")
-            raise ValueError(f"Invalid curve state: {str(e)}")
+            logger.error(f"Failed to get curve state: {e!s}")
```

Ruff (0.8.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)